### PR TITLE
📖 Update guidelines for runtime hook implementation

### PR DESCRIPTION
**Pull request by pravarag**
**What this PR does / why we need it**:
Updates guidelines for runtime hook implementation
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubernetes-sigs/cluster-api/issues/7812

### CODE REVIEW

**Comment by chrischdi** [comment]
/retitle 📖 Update guidelines for runtime hook implementation

File: docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md [hunk 1]
```diff
@@ -358,6 +358,8 @@ implementation documentation.
 
 ## Tips & tricks
 
+Make sure to add the ExtensionConfig object to YAML manifest used to deploy the runtime extensions (see [Extensionsconfig](#extensionconfig) for more details).
```
Comment (sbueringer) [inline docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md:361]:
```suggestion
Make sure to add the ExtensionConfig object to the YAML manifest used to deploy the runtime extensions (see [Extensionsconfig](#extensionconfig) for more details).
```
I think

**Comment by sbueringer** [comment]
Thx

/lgtm
/approve

**Pull request merged**